src/utils/localStorage.js

```diff
@@ -0,0 +1,25 @@
+
+const LOCAL_STORAGE_KEY = 'blogPosts';
+
+export const getPostsFromLocalStorage = () => {
+  const storedPosts = localStorage.getItem(LOCAL_STORAGE_KEY);
+  return storedPosts ? JSON.parse(storedPosts) : [];
+};
```

Comment:
add a parameter for `keyName` and make function generic

Reply:
done